<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QMainWindow" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SettingWindow{border-image: url(:/img/Resource/img/settingBackground.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1" rowspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>79</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QPushButton" name="pushButtonExitLogin">
      <property name="text">
       <string>退出/重新登录</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>81</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>143</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="difficultyWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
         <item>
          <widget class="QLabel" name="labelDifficult">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>难度:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ChooseByDirection" name="labelChooseDifficulty">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>143</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>143</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="gameModWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
         <item>
          <widget class="QLabel" name="labelgameMod">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>游戏模式:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ChooseByDirection" name="labelChoosegameMod">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>143</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>154</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1" colspan="2">
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,1,1">
      <item row="0" column="0" rowspan="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
        <item>
         <widget class="QCheckBox" name="checkBoxCustom">
          <property name="text">
           <string>自定义</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QWidget" name="customWidget" native="true">
        <layout class="QGridLayout" name="gridLayoutCustom">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <widget class="QLabel" name="labelRowNum">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>行数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxRowNum">
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>24</number>
             </property>
             <property name="value">
              <number>9</number>
             </property>
             <property name="number" stdset="0">
              <string>9</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="QLabel" name="labelColNum">
             <property name="text">
              <string>列数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxColNum">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="value">
              <number>9</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="labelBombNum">
             <property name="text">
              <string>雷数:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxBombNum">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="4" column="3">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>154</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>76</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="6">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="4">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>143</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="5" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChooseByDirection</class>
   <extends>QLabel</extends>
   <header location="global">ChooseByDirection.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
